app/views/includes/search-form.nunjucks

```diff
@@ -9,7 +9,7 @@
     <div class="form-item-wrapper">
       {% if errorMessage %}
         <h2>{{ searchErrorLabel }}</h2>
-        <p>{{ errorMessage }} </p>
+        <p>{{ errorMessage }}</p>
       {% elif gps.length === 0 %}
          <h2>We can't find a surgery using '{{search}}'</h2>
```

Comment:
Hey @neilbmclaughlin - I'm working on getting error messages sorted for out of England in the middleware, do you want me to look at 0 results too?